components/widget/widget_table.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Table
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Widget = require('Module:Infobox/Widget')
+local WidgetFactory = require('Module:Infobox/Widget/Factory')
+
+local Table = Class.new(
+	Widget,
+	function(self, input)
+		self.rows = input.rows or {}
+		self.classes = input.classes or {}
+		self.css = input.css or {}
+	end
+)
+
+function Table:addRow(row)
+	table.insert(self.rows, row)
+	return self
+end
+
+function Table:addClass(class)
+	table.insert(self.classes, class)
+	return self
+end
+
+function Table:make()
+	local table = mw.html.create('div')
+	table:css{display = 'inline-grid', ['border-right'] = '1px solid #bbb', ['border-bottom'] = '1px solid #bbb'}
```

Comment:
please move to css sheet we want to get rid of the hardcoded colors^^

Reply:
CSS will be into classes before the feature is considered live. 
However the outstanding functionality mentioned for upcoming PRs will likely require additional additional CSS changes, and I would rather move it to sheets once it's feature complete.